generate.js

```diff
@@ -60,8 +63,35 @@ const template = `
         </div>
       {{/orgs}}
     </div>
+    <script>
+      document.getElementById("last-generated").textContent = "Last generated " + timeDifference(new Date().getTime(), {{timestamp}})
+
+      function timeDifference(current, previous) {
+        const msPerMinute = 60 * 1000
+        const msPerHour = msPerMinute * 60
+        const msPerDay = msPerHour * 24
+        const msPerMonth = msPerDay * 30
+        const msPerYear = msPerDay * 365
+      
+        const elapsed = current - previous
+      
+        if (elapsed < msPerMinute) {
+          return Math.round(elapsed / 1000) + " seconds ago"
+        } else if (elapsed < msPerHour) {
+          return Math.round(elapsed / msPerMinute) + " minutes ago"
+        } else if (elapsed < msPerDay) {
+          return Math.round(elapsed / msPerHour) + " hours ago"
+        } else if (elapsed < msPerMonth) {
+          return "approximately " + Math.round(elapsed / msPerDay) + " days ago"
+        } else if (elapsed < msPerYear) {
+          return "approximately " + Math.round(elapsed / msPerMonth) + " months ago"
+        } else {
+          return "approximately " + Math.round(elapsed / msPerYear) + " years ago"
+        }
+      }      
+    </script>
```

Comment:
coala is going to be used soon, can you move the js to it's own file, preferably inside `static` folder.